src/Components/ExercisePage/Solution.tsx

```diff
@@ -1,3 +1,38 @@
+import { FileArrowUp, Plus } from 'phosphor-react';
+import { useRef } from 'react';
+
 export const Solution = () => {
-  return <p>Solution</p>;
+  const hiddenFileInput = useRef<HTMLInputElement>(null);
+
+  const handleClick = () => {
+    if (hiddenFileInput.current === null) return undefined;
+    hiddenFileInput.current.click();
+  };
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (!e.target.files) return undefined;
```

Comment:
What is the difference of "=== null" (like in the previous handle), and "!...files"? Also I think that if-statements should be extended to multiple lines with brackets and such.